test/integration/ctl/ctl_status_certificate_test.go

```diff
@@ -134,7 +143,9 @@ DNS Names:
 Events:  <none>
 Issuer:
   Name: letsencrypt-prod
-  Kind: ClusterIssuer
+  Kind: Issuer
```

Comment:
Given that it's valid for a Certificate to exist without a corresponding Issuer (albeit, not a particularly useful configuration), it'd be good to have tests that demonstrate and verify the output when the Issuer/ClusterIssuer **doesn't** exist too.